integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/events/AlterTableDropColumnEventListener.scala

```diff
@@ -60,7 +60,6 @@ class AlterTableDropColumnEventListener extends OperationEventListener {
       tablePath: String,
       alterTableDropColumnModel: AlterTableDropColumnModel)
     (sparkSession: SparkSession) {
-    var indexTableToDrop: Seq[String] = Seq.empty
```

Comment:
Please rename method name `dropApplicableSITables` to `checkIfDropColumnExistsInSI`